tests/test_primordial_f004_walk.py

```
@@ -0,0 +1,53 @@
+
+    # Get PBH parameters from the sampler (produces 'mass_msun', not 'mass')
+    pbh_params = generate_pbh_sample(1)[0]
+    # Verify the sampler produces the expected keys
+    assert 'mass_msun' in pbh_params
+    assert 'impact_param_au' in pbh_params
+    assert 'velocity_au_day' in pbh_params
+    assert 't_encounter_years' in pbh_params
+
+    # This should NOT raise KeyError: 'mass' - the runner should handle the sampler's keys
+    args = (positions, velocities, masses, t_start, t_end, dt, integrator, pbh_params)
+    times, positions_out, velocities_out = run_single_simulation(args)
```

Ruff (0.15.20)

[warning] Unpacked variable `positions_out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)